src/forms/stories/form.stories.js

```diff
@@ -6,10 +6,10 @@ import { action } from '@storybook/addon-actions';
 import { Form, Field } from 'react-final-form';
 import { Form as PfForm, Col, Row, Grid, Button } from 'patternfly-react';
 import { required, email } from 'redux-form-validators';
-import { FinalFormField, FinalFormSelect, FinalFormCheckBox, FinalFormRadio, FinalFormTextArea, Condition, composeValidators } from '../';
+import { FinalFormField, FinalFormSelect, FinalFormCheckBox, FinalFormRadio, FinalFormTextArea, FinalFormSwitch, Condition, FieldGroup, composeValidators } from '../';
```

Comment:
Wouldn't it be better to spread these imports over couple of lines so it might be a bit readable?
```javascript
import {
  FinalFormField,
  FinalFormSelect,
  FinalFormCheckBox,
  FinalFormRadio,
  FinalFormTextArea,
  FinalFormSwitch,
  Condition,
  FieldGroup,
  composeValidators
} from '../';

```